src/context/PlayerBoardContext.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, createContext, useEffect } from 'react';
+import axios from 'axios';
```

Comment:
This PlayerBoardContext looks exactly the same as the ComputerBoardContext.  The interfaces, although different names, are exactly the same.  Why do we need both?  Is there a way to rely on one?

Reply:
We refactored the boards to only use one context and one component now.